### PR TITLE
DM-18126 Support the nextVisit event

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: sal_interfaces/ScriptQueue/ScriptQueue_Events.xml [hunk 1]
```diff
@@ -61,7 +59,125 @@
...
+        <IDL_Type>double</IDL_Type>
+        <Units>degree</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>filters</EFDB_Name>
+        <Description>Comma-separated names of acceptable filters, or blank for any filter.</Description>
+        <IDL_Type>string</IDL_Type>
+        <IDL_Size>1</IDL_Size>
+        <Units>second</Units>
```
Comment (timj) [inline sal_interfaces/ScriptQueue/ScriptQueue_Events.xml]: Unitless I think.

File: sal_interfaces/ScriptQueue/ScriptQueue_Events.xml [hunk 2]
```diff
@@ -61,7 +59,125 @@
...
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>dome</EFDB_Name>
+        <Description>Desired dome state; a Script MetadataDome enum.</Description>
+        <IDL_Type>short</IDL_Type>
+        <Units>second</Units>
```
Comment (timj) [inline sal_interfaces/ScriptQueue/ScriptQueue_Events.xml]: Unitless?

File: sal_interfaces/ScriptQueue/ScriptQueue_Events.xml [hunk 3]
```diff
@@ -61,7 +59,125 @@
...
+        <IDL_Type>int</IDL_Type>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>survey</EFDB_Name>
+        <Description>Survey name.</Description>
+        <IDL_Type>string</IDL_Type>
+        <IDL_Size>1</IDL_Size>
+        <Units>second</Units>
```
Comment (timj) [inline sal_interfaces/ScriptQueue/ScriptQueue_Events.xml]: unitless

File: sal_interfaces/ScriptQueue/ScriptQueue_Events.xml (hunk 1 above)
Comment (r-owen) [reply sal_interfaces/ScriptQueue/ScriptQueue_Events.xml]: Thank you for catching these errors in units. I also fixed the same errors in the Script metadata event.

**Comment by r-owen** [comment]
I fixed the units and rebased to make the history more coherent. Could you please mark this reviewed one more time? Changing anything prevents merging until that is done.

**Pull request merged**